### PR TITLE
Features Documentation

### PR DESCRIPTION
**Pull request by Yawolf**
Documentation for features:
- [x] Transformation (translation)
- [x] Storage
- [x] Versioning

### CODE REVIEW

File: docs/docs/docs/features/translations.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ # Supported IDLs for Transformation
+ In this matrix you can see all the allowd transformations by Compendium so far:
+
+ | From/To  | Avro | Protobuf | Mu  | OpenAPI | Scala |
+ |----------|------|----------|-----|---------|-------|
+ | Avro     | -    | No       | Yes | No      | No    |
+ | Protobuf | No   | -        | Yes | No      | No    |
+ | Mu       | No   | No       | -   | No      | No    |
+ | OpenAPI  | No   | No       | No  | -       | No    |
+ | Scala    | No   | No       | No  | No      | -     |
```
Comment (JesusMtnez) [inline docs/docs/docs/features/translations.md:29]:
```suggestion
# Supported IDLs for Transformation

In this matrix you can see all the allowed transformations by Compendium so far:

| From/To  | Avro | Protobuf | Mu  | OpenAPI | Scala |
|----------|------|----------|-----|---------|-------|
| Avro     | -    | No       | Yes | No      | No    |
| Protobuf | No   | -        | Yes | No      | No    |
| Mu       | No   | No       | -   | No      | No    |
| OpenAPI  | No   | No       | No  | -       | No    |
| Scala    | No   | No       | No  | No      | -     |
```

File: docs/docs/docs/features/schema_storage.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Supported IDLs for Stroing
+These are the allowed IDLs that can be stored in Compendium so far:
+
+ | IDL      | Supported |
+ |----------|-----------|
+ | Avro     | Yes       |
+ | Protobuf | Yes       |
+ | Mu       | No        |
+ | OpenAPI  | No        |
+ | Scala    | No        |
```
Comment (JesusMtnez) [inline docs/docs/docs/features/schema_storage.md:31]:
```suggestion
# Supported IDLs for storing

These are the allowed IDLs that can be stored in Compendium so far:

| IDL      | Supported |
|----------|-----------|
| Avro     | Yes       |
| Protobuf | Yes       |
| Mu       | No        |
| OpenAPI  | No        |
| Scala    | No        |
```

**Comment by JesusMtnez** [comment]
@Yawolf Menu links should work now :ok_hand: 

Please, review all links in your documentation, they should never link to `MD` files directly, instead use a relative site link. Let me know if a could assists you more.

File: docs/docs/docs/features/storage/index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+For storing an schema you just need to do a `POST` request to this endpoint specifying the IDL in which your schema is written:
+```
+https://compendium-example.com/protocol/myProtocol?idlName=avro
+```
+
+# Versioning
+All the storaged schemas will be versioned. Currently the versioning is incremental. When a schema is storaged, the version of the schema is returned as a body response.
+
+# Supported IDLs for Stroing
```
Comment (juanpedromoreno) [inline docs/docs/docs/features/storage/index.md:21]:
```suggestion
# Supported IDLs for Storing
```

File: docs/docs/docs/features/translations/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (juanpedromoreno) [inline docs/docs/docs/features/translations/index.md:1]: Not sure about this section, but we could keep it for now and removing it later on..

Comment (Yawolf) [reply docs/docs/docs/features/translations/index.md:1]: Yeah, I decided to keep it because Compoendium, at the moment, supports it.

**Pull request merged**